localstripe/resources.py

```diff
@@ -244,6 +244,67 @@ def do_expand(path, obj):
         return obj
 
 
+class Balance(object):
+    object = 'balance'
+
+    def __init__(self, **kwargs):
+        if kwargs:
+            raise UserError(400, 'Unexpected ' + ', '.join(kwargs.keys()))
+
+        self.object = self.object
+        self.livemode = False
+        self.type = 'balance'
+        self.connect_reserved = []
+        self.available = {
+            'amount': 2000,
+            'currency': 'eur',
+            'source_types': {
+                'card': 2000
+            }
+        }
+        self.pending = {
+            'amount': 0,
+            'currency': 'eur',
+            'source_types': {
+                'card': 0
+            }
+        }
+
+        store[self.object] = self
+
+        schedule_webhook(Event('balance.available', self))
+
+    @classmethod
+    def _api_get(self):
+        obj = store.get(self.object, None)
+        if obj is None:
+            return self()
+        return obj
+
+    def _export(self, expand=None):
+        # expand is passed to be coherent with the server api
+        # but it's not used
```

Comment:
If it's not implemented, please drop it: it will make potentiel errors clearer.
Moreover, it seems to make no sense to pass `expand` to a Balance request.